src/platform/platform_internal.h

```diff
@@ -633,7 +634,9 @@ typedef struct CXPLAT_SOCKET {
     uint8_t Uninitialized : 1;
     uint8_t Freed : 1;
 
-    uint8_t UseTcp : 1;                  // Quic over TCP
+    uint8_t UseTcp : 1;  // This flag represents the expression (THIS SOCKET IS PART OF A CLIENT CONNECTION) && (THE CLIENT CONNECTION SPECIFIED TO USE QTIP)
```

Comment:
nit: This comment makes the line pretty long, it could go above the variable.
Could also be shorten "Set only for client connections using QTIP" or something similar?

Reply:
Should a similar comment be added on the matching variable for the linux definitions below?

Reply:
For the linux definition, that field should always be False.

Reply:
The "very long comment" part still needs to be addressed :)